packages/mediapipe-task-text/lib/src/mediapipe_text_bindings.dart

```diff
@@ -0,0 +1,292 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+// ignore_for_file: type=lint
+import 'dart:ffi' as ffi;
+import 'package:mediapipe_core/src/mediapipe_common_bindings.dart' as imp1;
```

Comment:
What is imp1 short for?

Reply:
I believe it's "implementation one", but this is generated code.